tensorvis/tensorvisio/include/modules/tensorvisio/tensorvisiomodule.h

```diff
@@ -35,12 +35,17 @@
 
 namespace inviwo {
 
+class VtkOutputLogger;
+
 class IVW_MODULE_TENSORVISIO_API TensorVisIOModule : public InviwoModule {
 public:
     TensorVisIOModule(InviwoApplication* app);
     virtual ~TensorVisIOModule() = default;
+
+private:
+    std::shared_ptr<VtkOutputLogger> vtkoutput_;
```

Comment:
Why shared pointer? Why not a regular member?

Reply:
Otherwise it is not possible to use a forward declaration. Will change to unique pointer.